main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/hashicorp/terraform/plugin"
+	"github.com/hashicorp/terraform/terraform"
+	"launchdarklytfprovider/launchdarkly"
+)
+
+func main() {
+	plugin.Serve(&plugin.ServeOpts{
+		ProviderFunc: func() terraform.ResourceProvider {
+			return launchdarkly.Provider()
```

Comment:
Tu peux juste faire `ProviderFunc: launchDarkly.Provider`

Reply:
Nah this doesn't compile. `Provider` returns a pointer, so the function is incompatible. I tried fiddling with that, but in the end this is copy pasta from the official Terraform doc so I'll leave it as-is.